cfme/tests/networks/test_provision_to_virtual_network.py

```diff
@@ -0,0 +1,74 @@
+from widgetastic.utils import partial_match
```

Comment:
Required: this should be ordered:
import fauxfactory
import pytest
from widgetastic.utils import partial_match